src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx

```diff
@@ -15,70 +15,104 @@ redirects:
 
-* From the [host filter](/docs/infrastructure/new-relic-infrastructure/filter-group/filter-sets-organize-your-infrastructure-hosts), select **Applications**.
-* Select the application you want to filter on.
+  <img
+    title="host with critical violations"
+    alt="A gif depicting a deeper inspection of a host with critical violations"
+    src={apmExploringaSpecificHostinAPM}
+  />
 
-  <Callout variant="tip">
-    On the **Hosts** page, you can also filter by selecting items in the **Applications** column.
-  </Callout>
+  <figcaption>
+    Inspect your host without leaving the APM summary page. 
+  </figcaption>
+  </Step>
+</Steps>
 
-## Switch between infrastructure and APM [#switch]
+Now that you know how to troubleshoot with APM and infrastructure monitoring we're going to explore how to integrate APM and infrastructure data and put it into practice. 
 
-When your <InlinePopover type="apm" /> and infrastructure accounts are linked, you can switch over from infrastructure to APM and vice versa for the same selected time range.
+## View logs for your APM and infrastructure data [#logs-context]
 
-You can switch from infrastructure to APM from these locations:
+You can also bring your logs and application's data together to make troubleshooting easier and faster. With [logs in context](/docs/logs/logs-context/configure-logs-context-apm-agents/), you can see log messages related to your errors and traces directly in your app's UI. You can also see logs in context of your [infrastructure data](/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent/), such as Kubernetes clusters. No need to switch to another UI page.
 
-* From the [host filter](/docs/infrastructure/new-relic-infrastructure/filter-group/filter-sets-organize-your-infrastructure-hosts) **Applications** menu
-* On the **Hosts** page, when selecting applications in the **Applications** table column.
+## Filter by application data [#filter-by-app]
 
-You can switch from APM to infrastructure from the host table on the APM [Summary](#infra-apm-overview) page.
+When your <InlinePopover type="apm" /> and infrastructure data is linked, you can filter displayed host data by searching for the specific application you want to inspect. In the case above, you would want to filter for `Billing Service`.
+
+<img
+    title="filter by application"
+    alt="A screenshot showing you how to filter the infrastructure page by application"
+    src={apmFilterbyApplication}
+/>
 
 ## APM data in Inventory and Events [#inventory-events]
```

Comment:
maybe 'APM data on inventory and events UI pages'. Don't think we can capitalize these (because we can't use bold in headers, and think about how that would look for UI pages with two or more words in name).